open-metadata-implementation/access-services/asset-lineage/asset-lineage-server/src/main/java/org/odpi/openmetadata/accessservices/assetlineage/handlers/AssetContextHandler.java

```diff
@@ -164,7 +164,7 @@ public RelationshipsContext buildAssetContext(String userId, EntityDetail entity
      * @throws OCFCheckedExceptionBase checked exception for reporting errors found when using OCF connectors
      */
     public Map<String, RelationshipsContext> buildColumnContext(String userId, String guid) throws OCFCheckedExceptionBase {
```

Comment:
This method builds the context for a TabularColumn/RelationalColumn, and doesn't need to be modified. The lineage mappings relationships are published in publishLineageMappingRelationshipEvent anyway(by calling publishLineageRelationshipEvent(lineageRelationship, eventType)) - so there is no need for this change